regolith/schemas.py

```diff
@@ -1131,6 +1132,11 @@
             "type": "list",
             "description": "other names for the person",
         },
+        'day': {
+            "description": "day when the entry was created/updated",
```

Comment:
which?  Created, or updated?  I think we want this to be the creation date, which is the first time we recognized that we knew this persion.  We have an updated field that can keep track of the most recent update.

Reply:
done!